rpc/src/test/scala/org/bitcoins/rpc/RawTransactionRpcTest.scala

```diff
@@ -0,0 +1,176 @@
+package org.bitcoins.rpc
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.bitcoins.core.config.NetworkParameters
+import org.bitcoins.core.currency.Bitcoins
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.script.{ P2SHScriptSignature, ScriptPubKey }
+import org.bitcoins.core.protocol.transaction.{ TransactionInput, TransactionOutPoint }
+import org.bitcoins.core.util.BitcoinSLogger
+import org.bitcoins.rpc.client.{ BitcoindRpcClient, RpcOpts }
+import org.scalatest.{ AsyncFlatSpec, BeforeAndAfterAll }
+import org.slf4j.Logger
+
+import scala.async.Async.{ async, await }
+import scala.concurrent.ExecutionContext
+
+class RawTransactionRpcTest extends AsyncFlatSpec with BeforeAndAfterAll {
+  implicit val system: ActorSystem = ActorSystem("RawTransactionRpcTest_ActorSystem")
+  implicit val m: ActorMaterializer = ActorMaterializer()
+  implicit val ec: ExecutionContext = m.executionContext
+  implicit val networkParam: NetworkParameters = TestUtil.network
+
+  val client = new BitcoindRpcClient(TestUtil.instance())
+  val otherClient = new BitcoindRpcClient(TestUtil.instance())
+
+  val logger: Logger = BitcoinSLogger.logger
+
+  override protected def beforeAll(): Unit = {
+    logger.info("Starting RawTransactionRpcTest")
+    logger.info("Bitcoin servers starting")
+    TestUtil.startServers(client, otherClient)
+    logger.info("Bitcoin servers started")
+  }
+
+  override protected def afterAll(): Unit = async {
```

Comment:
If I remember `scala-async` correctly you always need to wrap a `await` inside of a `async`. I don't think it makes sense to have this wrapped in `async`/`await`. We don't want `beforeAll`/`afterAll` executed asynchronously. Also, does this type check? I believe this returns a `Future[Unit]` not `Unit`?